src/functionalTest/java/uk/gov/hmcts/reform/cwrdapi/StaffRefUserTypeFunctionalTest.java

```diff
@@ -0,0 +1,101 @@
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestMethodOrder;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.http.HttpStatus;
+import org.springframework.test.context.ActiveProfiles;
+import uk.gov.hmcts.reform.cwrdapi.client.CaseWorkerApiClient;
+import uk.gov.hmcts.reform.cwrdapi.controllers.response.StaffRefDataUserTypesResponse;
+import uk.gov.hmcts.reform.cwrdapi.idam.IdamOpenIdClient;
+import uk.gov.hmcts.reform.cwrdapi.util.FeatureToggleConditionExtension;
+import uk.gov.hmcts.reform.cwrdapi.util.ToggleEnable;
+import uk.gov.hmcts.reform.lib.client.response.S2sClient;
+import uk.gov.hmcts.reform.lib.util.serenity5.SerenityTest;
+
+import javax.annotation.PostConstruct;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static uk.gov.hmcts.reform.cwrdapi.util.FeatureToggleConditionExtension.getToggledOffMessage;
+
+@ComponentScan("uk.gov.hmcts.reform.cwrdapi")
+@WithTags({@WithTag("testType:Functional")})
+@ActiveProfiles("functional")
+@SerenityTest
+@SpringBootTest
+@Slf4j
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
+class StaffRefUserTypeFunctionalTest extends AuthorizationFunctionalTest {
+
+    public static final String mapKey = "StaffRefDataController.fetchUserTypes";
+
+    protected static CaseWorkerApiClient caseWorkerApiClient;
+
+    @PostConstruct
+    public void beforeTestClass() {
+
+        SerenityRest.useRelaxedHTTPSValidation();
+
+        if (null == s2sToken) {
+            log.info(":::: Generating S2S Token");
+            s2sToken = new S2sClient(
+                    testConfigProperties.getS2sUrl(),
+                    testConfigProperties.getS2sName(),
+                    testConfigProperties.getS2sSecret())
+                    .signIntoS2S();
+        }
+
+        if (null == idamOpenIdClient) {
+            idamOpenIdClient = new IdamOpenIdClient(testConfigProperties);
+        }
+
+        caseWorkerApiClient = new CaseWorkerApiClient(
+                caseWorkerApiUrl,
+                s2sToken, idamOpenIdClient);
+    }
+
+    @SuppressWarnings("unchecked")
+    @Test
+    @ToggleEnable(mapKey = mapKey, withFeature = true)
+    void should_fetchUserTypes_200() {
+        StaffRefDataUserTypesResponse responses = (StaffRefDataUserTypesResponse)
+                caseWorkerApiClient.fetchUserType(
+                        HttpStatus.OK
+                );
+        assertTrue(responses.getUserTypes().size() > 0);
+    }
+
+    @Test
+    @ExtendWith(FeatureToggleConditionExtension.class)
+    @ToggleEnable(mapKey = mapKey, withFeature = false)
+    void should_fetchUserTypes_403_when_Api_toggled_off() {
+
+        Response response = caseWorkerApiClient.getMultipleAuthHeadersInternal(ROLE_CWD_SYSTEM_USER)
```

Comment:
You need to use "staff-admin" token

Reply:
@SujeethKumarAri - this test case is for 403.

Reply:
yes, you need to use the same token (staff-admin) as 200 scenario. The only difference is that you have toggled off the api.
If you use the token other than staff-admin then you will always get 403 whether you have toggled on/off.

Reply:
@SujeethKumarAri - updated